larray_editor/start.py

```diff
@@ -2,14 +2,23 @@
 
 import os
 import sys
+import inspect
 from larray_editor import edit
 
 
 def main():
     if os.name == 'nt':
         sys.stdout = open(os.devnull, "w")
         sys.stderr = open(os.path.join(os.getenv("TEMP"), "stderr-" + os.path.basename(sys.argv[0])), "w")
-    edit(display_caller_info=False)
+    try:
+        pos_arg = inspect.getfullargspec(edit).args.index('display_caller_info')
```

Comment:
Why do you need this code? 

My understanding after looking at the code for a few long minutes is this:
* the if part is there to avoid the code crashing if someone passes enough arguments via command-line to have a value for display_call_info via position in addition to via the keyword?
* I have no idea why you need the try/except.

I guess a few comments wouldn't hurt here.

Reply:
- Yes. But I'm not sure it worth to include the possibility that someone passes enough arguments via command-line to have a value for display_call_info via position. This is a bit extreme.
- The try/catch is because we may want one day to get rid of the display_call_info argument. 

Maybe calling edit(*sys.argv[:-1], display_call_info=False) is enough. What do you think?

Reply:
That's what I would have written in the first place. If we need to validate stuff more, we should use proper command line arguments parser and/or only ever accept a single string/filepath argument. Other arguments are mostly useless from the command line.